### PR TITLE
Add the -requestLengthPercentiles command-line flag

### PR DESCRIPTION
**Pull request by olix0r**
The strest-grpc client currently only sends very small request payloads,
including only the response spec. This change adds an arbitrary payload
to strest-grpc requests (that are ignored by the server) so that strest
clients can generate request payloads.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 strest-grpc*
 release/
+vendor/
```
Comment (klingerf) [inline .gitignore]: I think the need for a `vendor` directory was removed in #58, when we switched this repo to go modules. Is there something in your local env that's producing this directory? I don't see it when I run `GO111MODULE=on go run main.go` in this repo.

File: go.sum [hunk 2]
```diff
@@ -1,23 +1,137 @@
 cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
+github.com/BurntSushi/toml v0.3.1 h1:WXkYYl6Yr3qBf1K79EBnL4mak0OimBfB0XUf9Vl28OQ=
```
Comment (klingerf) [inline go.sum]: I'm also not totally sure how go modules work, but I don't think you need all of the additional changes to this file. What happens if you run `go mod tidy` on this branch?

File: .gitignore (hunk 1 above)
Comment (olix0r) [reply .gitignore]: I ran `go mod vendor` and it created this directory. I also don't really know what i'm doing with `go mod` so maybe it wasn't necessary...

Comment (klingerf) [reply .gitignore]: My understanding is that you only need the vendor directory if you're not running with go modules enabled. If you enable go modules, then that should handle fetching the dependencies automatically and storing them somewhere outside of your source tree. But yeah, idk

**Pull request merged**